packages/sdk/src/oracle/services/statemachines/setUser.ts

```diff
@@ -0,0 +1,39 @@
+import { Update } from "../update";
+import Store from "../../store";
+import { Signer } from "../../types/ethers";
+import { Handlers as GenericHandlers } from "../../types/statemachine";
+import { getFlags } from "../../utils";
+
+// require exports for a new context handler
+export type Params = {
```

Comment:
this is basically identical to the setActiveRequest handler, but wiht differnet params and slightly different handler logic